doc/redirects.json

```diff
@@ -317,6 +320,30 @@
   "sec-tools-of-stdenv": [
     "index.html#sec-tools-of-stdenv"
   ],
+  "ssec-cosmic-common-issues": [
```

Comment:
Not so sure about these changes. Is this actually something we add for new documentation or is this just for legacy reasons to not break old anchor tags?

Reply:
Do you see a build error if this is not present?

Reply:
CI was failing before I added redirects, so I think it is needed.